app/code/Magento/InventoryCache/Plugin/InventoryApi/CleanCacheAfterSourceItemsSavePlugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCache\Plugin\InventoryApi;
+
+use Magento\Framework\App\Cache\TypeListInterface as CacheTypeListInterface;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryApi\Api\SourceItemsSaveInterface;
+
+/**
+ * Clean cache after source items save plugin
+ */
+class CleanCacheAfterSourceItemsSavePlugin
+{
+    /**
+     * @var CacheTypeListInterface
+     */
+    private $cacheTypeList;
+
+    /**
+     * @param CacheTypeListInterface $cacheTypeList
+     */
+    public function __construct(CacheTypeListInterface $cacheTypeList)
+    {
+        $this->cacheTypeList = $cacheTypeList;
+    }
+
+    /**
+     * @param SourceItemsSaveInterface $subject
+     * @param void $result
+     * @param SourceItemInterface[] $sourceItems
+     * @return void
+     */
+    public function afterExecute(
```

Comment:
I believe we should add `@SuppressWarnings(PHPMD.UnusedFormalParameter)` here and in other plugins in order to pass through the static tests